### PR TITLE
Description and link to cosine added

### PR DESCRIPTION
**Pull request by AreloTanoh**
Added link to numpy cosine description 
#

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

File: sktime/transformations/series/cos.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 class CosineTransformer(_SeriesToSeriesTransformer):
     """Cosine transformation.
 
+    This is a wrapper around numpy's cosine function (see :func:`numpy.cosine`).
```
Comment (mloning) [inline sktime/transformations/series/cos.py:18]:
```suggestion
    This is a wrapper around NumPy's cosine function (see :func:`numpy.cos`).
```

Comment (mloning) [reply sktime/transformations/series/cos.py:18]: You can also add this section which will create a link to the numpy documentation: 

```
    See Also
    --------
    numpy.cos
```

**Review by mloning — COMMENTED** [review]
Hi @AreloTanoh, see my two quick comments below.

**Comment by AreloTanoh** [comment]
Changes added

**Review by mloning — APPROVED** [review]

**Pull request merged**